index.html

```diff
@@ -0,0 +1,116 @@
+        <a href="/login" class="nav-login-button"> 로그인 </a>
+      </nav>
+      <section class="banner-section">
+        <div class="banner-content">
+          <p class="banner-text">
+            일상의 모든 물건을<br />
+            거래해 보세요
+          </p>
+          <a href="/items" class="banner-button"> 구경하러 가기 </a>
+        </div>
+        <img src="assets/landing-page/banner.png" alt="banner-image" />
+      </section>
+    </header>
+    <main class="main-content">
+      <section>
+        <img
+          src="assets/landing-page/main-1.png"
+          alt="img-hot-item"
+          class="img-hot-item"
+        />
+        <article class="main-content-text">
+          <h2 class="category hot-item">Hot item</h2>
+          <h1 class="title hot-item">
+            인기 상품을<br />
+            확인해보세요
+          </h1>
+          <p class="description hot-item">
+            가장 HOT한 중고거래 물품을<br />
+            판다 마켓에서 확인해 보세요
+          </p>
+        </article>
+      </section>
+      <section>
+        <article class="main-content-text">
+          <h2 class="category search-item">Search</h2>
+          <h1 class="title search-item">
+            구매를 원하는<br />
+            상품을 검색하세요
+          </h1>
+          <p class="description search-item">
+            구매하고 싶은 물품은 검색해서<br />
+            쉽게 찾아보세요
+          </p>
+        </article>
+        <img src="assets/landing-page/main-2.png" alt="search-item" />
+      </section>
+      <section>
+        <img
+          src="assets/landing-page/main-3.png"
+          alt="img-hot-item"
+          class="img-hot-item"
+        />
+        <article class="main-content-text">
+          <h2 class="category hot-item">Hot item</h2>
+          <h1 class="title hot-item">
+            인기 상품을<br />
+            확인해보세요
+          </h1>
+          <p class="description hot-item">
+            가장 HOT한 중고거래 물품을<br />
+            판다 마켓에서 확인해 보세요
+          </p>
+        </article>
+      </section>
+    </main>
+    <footer>
+      <section class="banner-section">
+        <div class="banner-content">
+          <p class="banner-text">
+            믿을 수 있는<br />
+            판다마켓 중고 거래
+          </p>
+        </div>
+        <img src="assets/landing-page/footer.png" alt="footer-image" />
+      </section>
+      <section class="footer-section">
+        <div class="footer-copyright">©codeit - 2024</div>
+        <div class="footer-links">
+          <a href="/privacy">Privacy Policy</a>
+          <a href="/faq">FAQ</a>
+        </div>
+        <div class="footer-social">
+          <a href="https://www.facebook.com"
+            ><i class="fa-brands fa-facebook"></i
+          ></a>
+          <a href="https://www.x.com"><i class="fa-brands fa-twitter"></i></a>
+          <a href="https://www.youtube.com"
+            ><i class="fa-brands fa-youtube"></i
+          ></a>
+          <a href="https://www.instagram.com"
+            ><i class="fa-brands fa-instagram"></i
+          ></a>
+        </div>
```

Comment:
💊 제안
이 영역은 로고들의 모음이라고 할 수 있습니다. 관련된 요소들끼리 묶인 목록이니 의미록적인 마크업을 작성한다면 list 관련 태그로 작성하시는 것을 추천드립니다.

```suggestion
        <ul class="footer-social">
            <li>
              <a href="https://www.facebook.com">
                  <i class="fa-brands fa-facebook"></i>
              </a>
            </li>
        </ul>
```